src/visualizers/NumberGlyph.ts

```diff
@@ -403,11 +402,7 @@ The default value is 25.
     }
 }
 
-export const exportModule = new VisualizerExportModule(
-    'Number Glyphs',
-    NumberGlyph,
-    ''
-)
+export const exportModule = new VisualizerExportModule(NumberGlyph)
```

Comment:
Again, the others have a desc, maybe something like 'Colorful glyphs representing properties of the first few terms'